config.rb

```diff
@@ -25,8 +25,6 @@
   "2014/10/09/Distrubuted-Test-Execution.html" => "/2014/10/09/Distributed-Test-Execution/index.html",
   "2017/04/11/gocd-over-jenkins.html"          => "/2017/04/25/gocd-over-jenkins/index.html",
   "community/resources/index.html"             => "/resources/index.html",
-  "community/events/index.html"                => "/events/index.html",
-  "community/partners/index.html"              => "/partners/index.html",
```

Comment:
These were (likely) because of some external site which had that old link. But, I think it's fine to remove these two.

Reply:
Added back in 2aecd1551b7fc04e7f86b08b4452a58f881253b3, shouldn't have removed these .